.github/workflows/main-workflow.yml

```diff
@@ -13,14 +13,11 @@ jobs:
     - name: tshirts fail
       run: |
         g++ -Wall -Wextra -Werror tshirts.c -o tshirts.out
-        bash .github/workflows/expectfail.sh ./tshirts.out

```

Comment:
Removing this will miss the running of the tests. Retain `./tshirts.out` in the workflow. Same for the binaries below as well